setup.py

```diff
@@ -180,13 +180,17 @@ def _check_supported_system():
         )
 
     if sys.platform.startswith("darwin"):
-        # mac is not supported, raise error on install
-        raise OSError(
-            "Native Mac is currently unsupported for DeepSparse. "
-            "Please run on a Linux system or within a Linux container on Mac. "
-            "More info can be found in our docs here: "
-            "https://docs.neuralmagic.com/deepsparse/source/hardware.html"
-        )
+        if os.getenv("NM_ALLOW_DARWIN", "0"):
```

Comment:
If the env var is not set at all, won’t this return the string `"0"`, resulting in `if "0"` which is truthy/true?

Reply:
yup, should default to `False` or `None`.

```python
>>> bool("0")
True
```

Reply:
could also just set to `0`. just not str(0). confusing since env var "0" would be loaded as int and eval to False

Reply:
> confusing since env var "0" would be loaded as int and eval to False

I may be misunderstanding this comment, but environment variables are always strings, so `NM_ALLOW_DARWIN=0 python setup.py ...` would still yield the string `"0"` and "pass" that if conditional – it may ultimately be simpler to update the conditional to either `!= "0"` (open-ended) or `== "1"` (more strict).

Reply:
good catch. I'll update.

Reply:
thanks @dbarbuzzi and @bfineran ... I've updated accordingly